app/javascript/modules/EmailParliament/index.js

```diff
@@ -1,31 +1,22 @@
-// @flow
-// $FlowIgnore
 import React, { useState } from 'react';
 import classnames from 'classnames';
 import { search } from './api';
 import SearchByPostcode from './SearchByPostcode';
 import EmailComposer from './EmailComposer';
 import ComponentWrapper from '../../components/ComponentWrapper';
 import { redirect } from '../../util/redirector';
-import type { EmailPluginConfig } from '../../async/interfaces';
 
-export type Target = {
-  displayAs: string,
-  email: string,
-  gender: string,
-  id: string,
-  listAs: string,
-  name: string,
-  party: string,
-  picture: string,
+export const init = options => {
+  if (!options.config.active) return;
```

Comment:
is it worth checking presence of `config`?

Reply:
Yeah you're right